src/utils/v2/math.ts

```diff
@@ -0,0 +1,91 @@
+import { BigNumber } from '@ethersproject/bignumber'
+
+const TEN_THOUSAND = 10000
+const ONE_BILLION = 1000000000
+
+export const MAX_RESERVED_RATE = TEN_THOUSAND
+export const MAX_REDEMPTION_RATE = TEN_THOUSAND
+export const MAX_DISCOUNT_RATE = ONE_BILLION
+export const SPLITS_TOTAL_PERCENT = ONE_BILLION
+
+/**
+ * Express a given discount rate (parts-per-billion) as a percentage.
+ * @param discountRate - discount rate as parts-per-billion.
+ * @returns {string} discount rate expressed as a percentage.
+ */
+export const formatDiscountRate = (discountRate: BigNumber) => {
```

Comment:
My idea with these functions is that they end up being more clear, despite a bit of duplication.

Instead of trying to remember what units the discount rate etc. is in, we just use the `formatDiscountRate` function (and friends).

Reply:
Yep I like it SGTM